doc/api/assert.md

```diff
@@ -135,14 +136,7 @@ const assert = require('assert');
 ```
 
 Whenever possible, use the [`strict` mode][] instead. Otherwise, the
-[Abstract Equality Comparison][] may cause surprising results. This is
-especially true for [`assert.deepEqual()`][], where the comparison rules are
-lax:
-
-```js
-// WARNING: This does not throw an AssertionError!
-assert.deepEqual(/a/gi, new Date());
```

Comment:
We could change the example to use e.g.,

```js
assert.equal([[[1]]], true);
```

Reply:
I could do that if you feel strongly that the example bolsters the text. My thinking is that both the previous example and this one, while providing surprising results, also seem contrived and seem unlikely to occur. "Seem" is important there. It's all about reader perception. In other words, I wonder if the example actually reduces the sense of urgency to use `strict` mode. "Oh, I'd have to have something *that* unusual and unlikely before I ran into a problem? Whatever."

Reply:
@BridgeAR Clarification: That previous comment is how I'm feeling, but I also don't feel strongly. So if you feel strongly to the contrary, I'll put the example you suggest in there. Let me know.